scripts/config/cspell/dictionaries/qiskit.txt

```diff
@@ -158,11 +158,21 @@ mathrm
 exponentiated
 simulable
 resynthesizes
-sdist"
+sdist
 transpiling
 mthree
 Algorithmiq
 unphysical
 randomizations
 simulability
 unitarity
+misassignment
+toric
+gphase
+cobyla
+nelder
+stds
+accum
+optim
+synthes
+depolar
```

Comment:
These ones are word. We never use the shorthand, it's always the full word like `optimizing`. But cspell is cutting off the suffix and complaining about `optim`. Adding the full word `optimizing` to the dictionary didn't fix it.

Reply:
Weird design choice by cspell, thanks for the explanation though.